mpconstants/countries.py

```diff
@@ -184,6 +184,20 @@
         'special': False,
         'weight': 0
     },
+    'GAL': {
+        'adolescent': True,
+        'default_currency': 'EUR',
+        'default_language': 'gl',
+        'id': 4,
+        'legacy': 'GL',
```

Comment:
Sorry I was wrong, we do. I'm going to remove legacy in https://bugzilla.mozilla.org/show_bug.cgi?id=1122600.